### PR TITLE
Clean up Package.swift

### PR DESCRIPTION
**Pull request by mininny**
Use SPM distribution from SendBirdSDK instead of using the .xcframework file directly.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.5
+// swift-tools-version:5.3
```
Comment (jaesung-0o0) [inline Package.swift:1]: I'm gonna use features in Swift5.5. Reset this line plz

Comment (mininny) [reply Package.swift]: Done!

**Comment by jaesung-0o0** [comment]
@mininny  I could see some files removed. Did you remove `.xcframework` file in the directory?

**Comment by mininny** [comment]
> @mininny I could see some files removed. Did you remove `.xcframework` file in the directory?

I removed the whole .xcframework file

**Comment by jaesung-0o0** [comment]
>> I removed the whole .xcframework file 

Thanks. This branch will be merged into develop and be involved in 0.2.0 release.

**Review by jaesung-0o0 — APPROVED** [review]
LGTM

**Pull request merged**